src/routes/choice-exercises/components/SingleChoiceExercise.tsx

```diff
@@ -148,7 +149,7 @@ const ChoiceExerciseScreen = ({
         numberOfWords={count}
         feedbackType={FeedbackType.document}
         feedbackForId={document.id}
-        exerciseKey={exerciseKey}
+        exerciseKey={ExerciseKeys.wordChoiceExercise}
```

Comment:
Why did you change that from `exerciseKey` to `ExerciseKeys.wordChoiceExercise`?
```suggestion
        exerciseKey={exerciseKey}
```

Reply:
This looks really wrong though. SingleChoiceExercise may also be ArticleChoiceExercise in which case your current solution would pass the wrong exercise key.